main/openssh/APKBUILD

```diff
@@ -29,16 +28,6 @@ source="http://ftp.openbsd.org/pub/OpenBSD/OpenSSH/portable/$pkgname-$_pkgver.ta
 	"
 builddir="$srcdir/$pkgname-$_pkgver"
 
-# secfixes:
-#   7.4_p1:
-#     - CVE-2016-10009
-#     - CVE-2016-10010
-#     - CVE-2016-10011
-#     - CVE-2016-10012
-#       https://bugs.alpinelinux.org/issues/6583
-
-# HPN patches are from: http://www.psc.edu/index.php/hpn-ssh
-
```

Comment:
We need to keep this comment. It is used to create a whitelist for a CVE scanner.